### ISSUE TITLE
Wrong attributes access in 'R0Container'

### ISSUE THREAD

**Issue by mavillan** [cta-observatory/image-extractor#17]
Hi,

When running image-extractor over a simtel file: `gamma_20deg_0deg_run8___cta-prod3_desert-2150m-Paranal-HB9_cone10.simtel` I have this error: 

```
Traceback (most recent call last):
  File "image_extractor.py", line 641, in <module>
    extractor.process_data(data_file, args.max_events)
  File "image_extractor.py", line 462, in process_data
    event_row['run_number'] = event.r0.run_id
AttributeError: 'R0Container' object has no attribute 'run_id'
Closing remaining open files:./dataset.hdf5...done
```

I temporarily bypassed it by commenting the line 462. However, I'm not sure if this might cause trouble when reading/using the hdf5 produced.

**Comment by Legutier** [comment]
The run_id attribute has changed to obs_id in the event object, a change in the code is needed. Try this and it should work just fine.

**Comment by bryankim96** [comment]
Thanks Lukas. This has now been addressed with the merging of your PR #18 in https://github.com/cta-observatory/image-extractor/commit/efe92aedc2ea422c4e3e1985061f7666c9f529e5.
